.travis.yml

```diff
@@ -0,0 +1,19 @@
+sudo: false
+
+language: python
+
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+
+cache:
+  directories:
+      - $HOME/.cache/pip
+
+install:
+    - pip install travis-sphinx sphinx
+    - pip install -e . 
+
+script:
+    - python setup.py test
```

Comment:
If you can't get the Travis CI to work just move these changes to a different branch so that we can figure this out later on.